subsys/net/lib/lwm2m/README_lwm2m

```diff
@@ -0,0 +1,50 @@
+LwM2M Client in Zephyr
+----------------------
+
+### SenML CBOR
+
+SenML CBOR content format support takes advantage of the Zcbor library's capability of generating
+domain specific code. To generate the SenML encoder and decoder the following CDDL file
```

Comment:
I imagine a future you or me will be glad if you made these instructions into a bash script instead :).